README.rst

```diff
@@ -334,7 +334,7 @@ Call :code:`env.sh` script to set the test connection parameters in the environm
 
 .. code-block:: bash
 
-    source ./scripts/env.sh
+    ./scripts/env.sh && env | grep SNOWFLAKE_TEST > testenv.ini
```

Comment:
Removed `source` because in env.sh [the line`$( cd "$DIR/.." && pwd)/parameters.json`](https://github.com/snowflakedb/pdo_snowflake/blob/master/scripts/env.sh#L9) from the project root dir would put us outside the project